index.html

```diff
@@ -146,6 +146,43 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________Dex's card END ________  -->
 
```

Comment:
Please leave just one empty line before and one empty line after your code.

Reply:
HI @dilshankarunarathne  I have corrected this .. Please kindly review